drivers/virtualbox/virtualbox.go

```diff
@@ -417,6 +466,16 @@ func (d *Driver) Start() error {
 		log.Infof("VM not in restartable state")
 	}
 
+	// Verify that VT-X is not disabled in the started VM
+	disabled, err := d.IsVTXDisabledInTheVM()
+	if err != nil {
+		return err
```

Comment:
Please wrap this in another error, e.g.:

```golang
return fmt.Errorf("Checking if hardware virtualization is enabled failed: %s", err)
```

It will be easier to track down an issue if something fails that way.